modules/apps/27-interchain-accounts/controller/types/params_legacy.go

```diff
@@ -27,7 +27,7 @@ func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
 	}
 }
 
-func validateEnabledTypeLegacy(i interface{}) error {
+func validateEnabledTypeLegacy(i any) error {
```

Comment:
which linter requires this? Is there any benefit here?

Reply:
not much, `any` is a type alias of `interface{}`. It really makes no difference but we could choose either/or.

Reply:
use-any

it can be disabled, if we'd like.